modules/tinymce/src/plugins/table/test/ts/browser/command/ApplyCellStyleCommandTest.ts

```diff
@@ -11,6 +13,24 @@ UnitTest.asynctest('browser.tinymce.plugins.table.command.ApplyCellStyleCommandT
   Plugin();
   SilverTheme();
 
+  let events = [];
+  const logEvent = (event: EditorEvent<{}>) => {
+    events.push(event);
+  };
+
+  const sClearEvents = () => Step.sync(() => events = []);
```

Comment:
This could just be a hardcoded step, it doesn't need to be thunked.
```suggestion
  const sClearEvents = Step.sync(() => events = []);
```